src/app/components/top/solicitudes/detalleSolicitud.html

```diff
@@ -20,16 +20,14 @@
         </div>
         <div class="col-12 text-left">
             <b>Paciente</b>
-            <h5>
-                <span class="blue"> {{prestacionSeleccionada?.paciente?.nombre}}
-                    {{prestacionSeleccionada?.paciente?.apellido}}
-                </span>
-            </h5>
+            <br><span class="blue"> {{prestacionSeleccionada?.paciente?.nombre}}
```

Comment:
Acomodar las lineas, así evitamos en el futuro conflictos con linteos. 
```suggestion
            <br>
                <span class="blue"> 
                     {{prestacionSeleccionada?.paciente?.nombre}}
```

Reply:
corregido